framework/test/unit/application.spec.ts

```diff
@@ -58,6 +58,12 @@ describe('Application', () => {
 	});
 
 	describe('#constructor', () => {
+		it('should be able to start the application with default parameters if config is not provided', () => {
+			const app = Application.defaultApplication(genesisBlockJSON);
+
+			expect(app.config).toBeDefined();
```

Comment:
i think it would be nice to check if the optional configs (like minRemainingBalance) is set